### PR TITLE
withCustomFields method is added for SessionTokenRequest.

### PR DESCRIPTION
**Pull request by sercanparker**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -78,6 +78,12 @@
 			<version>1.10.0</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>jakarta.xml.bind</groupId>
+			<artifactId>jakarta.xml.bind-api</artifactId>
+			<version>2.3.3</version>
+			<scope>compile</scope>
+		</dependency>
 	</dependencies>
```
Comment (isahb) [inline pom.xml]: Why do we need this?

File: pom.xml [hunk 2]
```diff
@@ -78,6 +78,12 @@
 			<version>1.10.0</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>jakarta.xml.bind</groupId>
+			<artifactId>jakarta.xml.bind-api</artifactId>
+			<version>2.3.3</version>
+			<scope>compile</scope>
+		</dependency>
```
Comment (isahb) [inline pom.xml]: Why do we need this?

Comment (sercanparker) [reply pom.xml]: I added due to this error while working on my local. But, it does not effect to my side that's why i removed it in my last commit :) 
image

**Review by isahb — APPROVED** [review]
Thanks

**Pull request merged**